### PR TITLE
[static-build] Apply Gatsby default caching routes automatically

### LINKED ISSUE #3331: Default Gatsby configuration provides no cache options
Hello!

I have just deployed a static gatsby page using your service and basically none of the static content is cached.
Video `mp4` cache: `cache-control: s-maxage=0`
images etc
```
cache-control: public, max-age=0, must-revalidate
```

I see this MR probably breaks it - https://github.com/zeit/now/pull/3212

I will use for now https://github.com/universse/gatsby-plugin-zeit-now

### PR DESCRIPTION
**Pull request by pacocoursey**
CH-702
Fixes #3331

If the file generated by `gatsby-plugin-now` does not exist, we'll set up these routes automatically.

### CODE REVIEW

File: packages/now-static-build/src/frameworks.ts [hunk 1]
```diff
@@ -42,26 +40,33 @@ const frameworkList: Framework[] = [
     dependency: 'gatsby',
     buildCommand: 'gatsby build',
     getOutputDirName: async () => 'public',
-    defaultRoutes: async (dirPrefix: string) => {
-      try {
-        const nowRoutesPath = join(
-          dirPrefix,
-          'public',
-          '__now_routes_g4t5bY.json'
```
Comment (styfle) [inline packages/now-static-build/src/frameworks.ts]: I think we might need to keep this file for backwards compatibility.

@lucleray What do you think?

Comment (lucleray) [reply packages/now-static-build/src/frameworks.ts]: It's not used by a lot of users, but I'd still keep it 👍

Comment (lucleray) [reply packages/now-static-build/src/frameworks.ts]: It's used in:
https://www.npmjs.com/package/gatsby-plugin-now
https://www.npmjs.com/package/gatsby-plugin-zeit-now

File: packages/now-static-build/src/frameworks.ts [hunk 2]
```diff
@@ -42,26 +40,33 @@ const frameworkList: Framework[] = [
...
-        const nowRoutes = JSON.parse(content);
-        try {
-          await unlinkPromise(nowRoutesPath);
-        } catch (err) {
-          // do nothing if deleting the file fails
-        }
-        return nowRoutes;
-      } catch (err) {
-        // if the file doesn't exist, we don't create routes
-        return [];
```
Comment (lucleray) [inline packages/now-static-build/src/frameworks.ts:62]: I suggest returning the default routes you added here, and keep the old behavior if `__now_routes_g4t5bY.json` exists.

**Review by lucleray — CHANGES_REQUESTED** [review]
Can you add a test for the headers? You might be able to copy the tests we had before: https://github.com/vercel/vercel/pull/3308/files#diff-83715bd09c6fbf83efb4fc7fac8f51f1R16-R58

**Review by lucleray — APPROVED** [review]
Thanks!

**Review by styfle — APPROVED** [review]
Thanks, nice work!

**Pull request merged**